builtin/mainmenu/tab_settings.lua

```diff
@@ -346,6 +342,9 @@ local function formspec(tabview, name, tabdata)
 			"checkbox[3.85,4.3;cb_vrange_auto_adjust;" .. fgettext("Auto-adjust") .. ";" ..
 				dump(core.setting_getbool("viewing_range_auto")) .. "]"
 	end
+	if get_vrange_min > get_vrange_max then
```

Comment:
Okay, fair enough.  But what if a user accidentally sets it to a string value instead?